### PR TITLE
[IMP] stock,delivery: Report improvement.

### PR DESCRIPTION
**Pull request by ppr-odoo**
Pad:https://pad.odoo.com/p/r.cb1f43da6b18e8b1db4bebba01f22b99
Task:https://www.odoo.com/web#id=1889404&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720

### CODE REVIEW

**Comment by amoyaux** [comment]
@ppr-odoo could you rebase on master? It seems that the installation of batch picking fails with demo data.

**Comment by ppr-odoo** [comment]
Hello  @amoyaux  rebase the branch please check it

File: addons/delivery/views/report_deliveryslip.xml [hunk 1]
```diff
@@ -1,21 +1,37 @@
...
+        <xpath expr="//div[@name='div_sched_date']" position="after">
+            <div t-if="o.picking_type_id.code == 'outgoing' and o.carrier_id" class="col-auto">
+                <strong>Carrier:</strong>
+                <p t-field="o.carrier_id"/>
+            </div>
+            <div t-if="o.shipping_weight" class="col-auto">
+                <strong>Weight:</strong>
+                <br/>
                 <span t-field="o.shipping_weight"/>
                 <span t-field="o.weight_uom_name"/>
```
Comment (sle-odoo) [inline addons/delivery/views/report_deliveryslip.xml]: @amoyaux do we need _get_weight_uom_id_from_ir_config_parameter here?

**Comment by sle-odoo** [comment]
robodoo r+ rebase-ff

**Pull request closed without merging**